frontend/src/components/App.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import axios, { AxiosResponse } from 'axios';
+import PractitionerFeedListItem from './PractitionerFeedListItem';
+import { HapiPractitionerEntryItem, HapiPractitionersResponse } from '../types/Hapi.types';
+import { getPractitionerEmail } from '../utils/helpers';
+
+function App() {
+  const [practitioners, setPractitioners] = useState<HapiPractitionerEntryItem[]>([]);
+
+  useEffect(() => {
+    axios.get<HapiPractitionersResponse>('http://hapi.fhir.org/baseDstu3/Practitioner')
```

Comment:
1. not reason. just a preference.
2. i couldn't use async in the useEffect unless i created some async function and then call it so i thought it would be easier to just use a promise. the next refactor i'd do is to move all the api calls into it's own service and use async/await there.